cypress/integration/specialFeatures/cookieBanner/testsForAMPOnly.js

```diff
@@ -81,6 +81,13 @@ Object.keys(config)
 
                 getCookieBanner(service, variant).should('not.be.visible');
                 getPrivacyBanner(service, variant).should('not.be.visible');
+
+                cy.on('uncaught:exception', err => {
```

Comment:
Hey, I know this is a draft, but I wondered if this might be a simpler way to fix this:

We already have a place for uncaught exceptions, and it could be a matter of just adding another line here?
https://github.com/bbc/simorgh/blob/latest/cypress/support/index.js#L17

Reply:
That seems like the right place!!

Reply:
I'd missed that :)

Reply:
I think we can replace that line, I'll update the test and push this up again!